tokio-trace-beacon/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "tokio-trace-beacon"
```

Comment:
Personally, I don't find this name particularly descriptive. When I saw this PR title, I had some pretty different assumptions about what `tokio-trace-beacon` was going to be. Can we come up with something a little more obvious?

Reply:
It might be worth having a `tokio-trace-test` crate, or similar. That way, we can combine this code for testing subscribers with the mock subscriber that's used in `tokio-trace` and `tokio-trace-futures` for testing.